network-api/networkapi/templates/pages/buyersguide/article_page.html

```diff
@@ -0,0 +1,57 @@
+{% extends "pages/buyersguide/base.html" %}
+
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block category_nav %}{% endblock category_nav %}
+
+{% block hero %}
+  {% if page.hero_image %}
+    {% image page.hero_image max-200x200 %}
+  {% endif %}
+{% endblock hero %}
+
+{% block guts %}
+  <div>
+    <h2>Categories</h2>
+    <ul>
+      {% for content_category_relation in page.content_category_relations.all %}
+        <li>{{ content_category_relation.content_category.title|title }}</li>
+      {% endfor %}
+    </ul>
+  </div>
+  <h1>{{ page.title }}</h1>
```

Comment:
**Question**

This might be an exsisting pattern, but is this heading order bad for SEO ?

h2 categories
h1 page title
h2 authors

Reply:
Yup total slip. Sorry. That's just bad for semantics and thus SEO and accessibility. 

Fixed in 0bc97204ae39400615c23f659c5318035710b592